java/flight/flight-sql-jdbc-core/src/main/java/org/apache/arrow/driver/jdbc/accessor/impl/calendar/ArrowFlightJdbcTimeStampVectorAccessor.java

```diff
@@ -91,14 +92,11 @@ private LocalDateTime getLocalDateTime(Calendar calendar) {
     long value = holder.value;
 
     LocalDateTime localDateTime = this.longToLocalDateTime.fromLong(value);
+    ZoneId defaultTimeZone = TimeZone.getDefault().toZoneId();
```

Comment:
I think we should avoid the default time zone here, since that will be system dependent?

Reply:
So the idea here is that if no calendar is supplied, we will use the system timezone as an assertion of what the timezone of the timestamp value from the db is. So during the accessor process, if 2021-03-28 00:15 is contained in the DB and we don't give the getter a calendar, we will then assume 2021-03-28 00:15 is an instant at the system time zone. Given that since the timestamp is already in the local timezone, no conversion is needed and the value returned by the getter is indeed 2021-03-28 00:15 (LOCAL TIMEZONE)